Common/CommonTester/BaseView/BaseView/PropertiesView.cs

```diff
@@ -69,5 +70,21 @@ private void buttonSave_Click(object sender, EventArgs e)
             buildOptions.Save();
             this.Close();
         }
+
+        private void splitContainerProperties_Panel2_Paint(object sender, PaintEventArgs e)
+        {
+
+        }
+       
+
+        private void pictureBox1_Click_1(object sender, EventArgs e)
```

Comment:
please change the names to meaningful names

Reply:
the name was changed